lib/postprocessing/image.js

```diff
@@ -39,16 +39,23 @@ const SUPPORTED_OUTPUT_FMT = {
 // Note: order is important and defines priority for intermediate format - first entry will be most preferred
 const SUPPORTED_INPUT_TYPES = {
     "TIFF": "tif",  // most preferred format as it allows to preserve many features
-    "PNG":  "png",  // lossless over
+    "PNG": "png",  // lossless over
     "JPEG": "jpg",  // ..lossy
-    "GIF":  "gif",  // (should never get here in practice)
-    "BMP":  "bmp"
+    "GIF": "gif",  // (should never get here in practice)
+    "BMP": "bmp",
+    "SVG": "svg"
```

Comment:
Would be nice to keep the "column/table" style indentation I added for readability.

Reply:
(done)